### ISSUE TITLE
Stage alpha property not implemented

### ISSUE THREAD

**Issue by meibegger** [bp74/StageXL#208]
stage.alpha = NUM;
> Exception: Unsupported operation: The Stage class does not implement this property or method.package:stagexl/src/display/stage.dart:286 Stage._throwStageException@822385374package:stagexl/src/display/stage.dart:299 Stage.set:alpha

**Comment by bp74** [comment]
Yes that may be a useful extension. For now, you can just use a display object instead:

    var myRoot = new Sprite();
    myRoot.alpha = 0.5;
    myRoot.addTo(stage);

**Comment by meibegger** [comment]
it's in the specs, if i understand correctly - so maybe just remove it ... 
http://www.stagexl.org/docs/api/index.html#stagexl/stagexl-display.Stage@id_align=

**Comment by bp74** [comment]
The reason is that Stage extends a normal DisplayObject which has all the normal properties like x,y,rotation and of course alpha. But changing most of those properties doesn't make sense for the Stage - for example changing the rotation of the Stage is strange. Therefore all getters and setters to those properties throw an exception. This is the exception you saw for the alpha setter too. But if you t think about it, setting the alpha of the Stage makes sense and therefore we could remove this limitation from the alpha-setter and getter.

https://github.com/bp74/StageXL/blob/master/lib/src/display/stage.dart#L290
